src/en/manhwabuddy/src/eu/kanade/tachiyomi/extension/en/manhwabuddy/ManhwaBuddy.kt

```diff
@@ -0,0 +1,138 @@
+package eu.kanade.tachiyomi.extension.en.manhwabuddy
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ManhwaBuddy : ParsedHttpSource() {
+    override val baseUrl = "https://manhwabuddy.com/"
+    override val lang = "en"
+    override val name = "ManhwaBuddy"
+    override val supportsLatest = true
+
+    override fun chapterFromElement(element: Element): SChapter {
+        return SChapter.create().apply {
+            setUrlWithoutDomain(element.attr("abs:href"))
+            name = element.selectFirst(".chapter-name")!!.text()
+            date_upload = element.selectFirst(".ct-update")?.text()?.let { parseDate(it) }!!
+        }
+    }
+
+    private fun parseDate(dateStr: String): Long {
+        return try {
+            dateFormat.parse(dateStr)!!.time
+        } catch (_: ParseException) {
+            0L
+        }
+    }
+
+    private val dateFormat by lazy {
+        SimpleDateFormat("dd MMMM yyyy", Locale.ENGLISH)
+    }
+
+    override fun chapterListSelector(): String {
+        return ".chapter-list a"
+    }
```

Comment:
Functions that return a string can generally be simplified.
```suggestion
    override fun chapterListSelector(): String = ".chapter-list a"
```

Comment also applies to similar patterns used throughout this file.